dataland-internal-storage/src/test/kotlin/db/migration/V4__MigrateEuTaxonomyNamesTest.kt

```diff
@@ -0,0 +1,75 @@
+package db.migration
+
+import db.migration.utils.buildDatabaseEntry
+import db.migration.utils.mockAndWhenConfigurationForFrameworkMigration
+import org.dataland.datalandbackend.openApiClient.model.DataTypeEnum
+import org.dataland.datalandbackend.openApiClient.model.YesNo
+import org.flywaydb.core.api.migration.Context
+import org.json.JSONObject
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito
+
+class V4__MigrateEuTaxonomyNamesTest {
+
+    @Test
+    fun `test that eu taxonomy for non financials migration script works as expected`() {
+        testIfDataIsMaintained(YesNo.yes, YesNo.no, DataTypeEnum.eutaxonomyMinusNonMinusFinancials)
+    }
+
+    @Test
+    fun `test that eu taxonomy for financials migration script works as expected`() {
+        testIfDataIsMaintained(YesNo.yes, YesNo.no, DataTypeEnum.eutaxonomyMinusFinancials)
+    }
+
+    @Test
+    fun `test that eu taxonomy migration script works as expected with unprovided values`() {
+        testIfDataIsMaintained(YesNo.yes, null, DataTypeEnum.eutaxonomyMinusNonMinusFinancials)
```

Comment:
Zum Verständnis: Wo werden die beiden gemockten Einträge denn verglichen?

Reply:
Scheinbar in `mockAndWhenConfigurationForFrameworkMigration`.  Ist aber auch Neuland für mich. Habe bislang noch nicht an unit tests für migration scripts gearbeitet.

Reply:
So habe ich das auch gedacht. Ich schau gerade mal tiefer rein.

Reply:
Ich schaue nochmal ein bisschen rein, mache hier aber schon mal zu und frage ansonsten Florian am Montag wie genau das funktioniert.